cmd/platform/user.go

```diff
@@ -164,7 +163,7 @@ func userActivateCmdF(cmd *cobra.Command, args []string) error {
 	}
 
 	if len(args) < 1 {
-		return errors.New("Enter user(s) to activate.")
+		return errors.New("Expected at least one argument. See help text for details.")
```

Comment:
Discussed above with @jasonblais. Some of the documentation was out of date or didn't match the --help text. My goal was to make a single source of information that users can rely on for the correct way to interpret errors.